src/main/java/io/vantiq/client/Vantiq.java

```diff
@@ -488,6 +488,31 @@ public VantiqResponse select(String resource,
                                  Object where,
                                  SortSpec sortSpec,
                                  long limit) {
+        return this.select(resource,propSpecs,where,sortSpec,0L,null);
```

Comment:
@steve-langley, there is a typo here. 
shouldn't this be "limit, null" instead of "0L, null"  ??

Reply:
Yes, you're right, thanks for catching this. Copy/paste error...

Reply:
and maybe add a test for that case?